infra/aws/terraform/iam/eks-e2e-boskos/cloudformation-role.json

```diff
@@ -0,0 +1,67 @@
+{
+  "Resources": {
+    "Provisioner": {
+      "Type": "AWS::IAM::Role",
+      "Properties": {
+        "RoleName": "Provisioner",
+        "AssumeRolePolicyDocument": {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Action": "sts:AssumeRole",
+              "Effect": "Allow",
+              "Principal": {
+                "AWS": "arn:aws:iam::995654820765:user/provisioner"
```

Comment:
Try something like:
```yaml
                "AWS": "arn:${AWS::Partition}:iam::${SomeNamedVariable}:user/provisioner"
```

Reply:
`${AWS::Partition}` substitution didn't work, but Account ID is now a parameter.